### PR TITLE
Node 6

### PR DESCRIPTION
**Pull request by jonstacks**
I think this does the trick, but its very possible it breaks things not covered by tests.

Re-Open of #156 since travis settings weren't configured correctly.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -27,7 +27,4 @@ after_script:
 
 language: node_js
 node_js:
-  - "4"
-  - "5.6"
-  - "5.12"
```
Comment (dustinsmith1024) [inline .travis.yml]: lets keep these around

Comment (dustinsmith1024) [reply .travis.yml]: or at least 5.12

Comment (jonstacks) [reply .travis.yml]: @dustinsmith1024, 5.12 doesn't have `Proxy` which this implementation will need. I am hoping that we can update the major version and only support node-6 forward.

**Comment by jonstacks** [comment]
@dustinsmith1024, any objections if we release this with a major version bump. I'm sure there will be a few small bug fixes, but it might be nice to get this released for easier testing on node apps >= 6.

**Comment by dustinsmith1024** [comment]
Nope. Do you need me to release or do you want to do it?

File: lib/proxy.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+
+function methodMissing(obj, handler) {
+    
+    const proxyObject = new Proxy(obj, {
+        get(object, property) {
+            if (Reflect.has(object, property)) {
+                return Reflect.get(object, property);
+            } else {
+                return handler.call(object, property);
+                // return (...args) => Reflect.apply(handler, proxyObject, [property, ...args]);
```
Comment (dylanstanfield) [inline lib/proxy.js]: remove

Comment (jonstacks) [reply lib/proxy.js]: Fixed in eeaee8b

**Comment by jonstacks** [comment]
@dustinsmith1024, If you could do it, that would be great.

**Comment by dustinsmith1024** [comment]
Ready to go. https://www.npmjs.com/package/patio/v/3.0.0.

I had some issues generating the docs. Did anything change that needs a doc update?

**Comment by jonstacks** [comment]
@dustinsmith1024, did you get any specific error that you can share. I changed a few things to classes but I didn't really have to change much code.

**Pull request merged**